fixture-monkey-datafaker/src/main/java/com/navercorp/fixturemonkey/datafaker/arbitrary/BaseStringCombinableArbitrary.kt

```diff
@@ -0,0 +1,69 @@
+package com.navercorp.fixturemonkey.datafaker.arbitrary
```

Comment:
/*
 * Fixture Monkey
 *
 * Copyright (c) 2021-present NAVER Corp.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */


You should add the license for all files.